packages/npm/@sentry/svelte/7.9.0.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Sentry Svelte SDK",
+  "canonical": "npm:@sentry/svelte",
+  "version": "7.9.0",
+  "package_url": "https://npmjs.com/package/@sentry/svelte",
+  "repo_url": "https://github.com/getsentry/sentry-javascript",
+  "main_docs_url": "https://docs.sentry.io/platforms/javascript/"
```

Comment:
We're currently not planning on having official docs for the first Alpha release of the Svelte SDK. I linked to the general Javascript (= Browser SDK) guide for the time being. The Svelte SDK right now is only a wrapper around the Browser SDK without much added stuff on top (that will follow later on). Is this fine?

Reply:
We can just put a link to where svelte will live in the future. The things that consume release registry (the docs and sentry onboarding flows) are not yet going to exist yet, so we can do this without any problems.

Reply:
I see, then I'll change the link to the future Svelte guide. My concern was that this would already somewhere be shown in the UI and hence point to a link that doesn't (yet) exist.